src/events/messageCreate.js

```diff
@@ -16,36 +16,37 @@ async function messageCreateHandler(message) {
 		return;
 	}
 
-	// * ayy => lmaoo
-	if (ayyRegex.test(message.content)) {
-		const lmaod = message.content.replaceAll(ayyRegex, (match) => {
-			let newMatch = match.replaceAll('y', 'o').replaceAll('Y', 'O');
-			newMatch = newMatch.replaceAll('a', 'lma').replaceAll('A', 'LMA');
-			return newMatch;
-		});
-
-		// Check that the message isn't too long to be sent
-		if (lmaod.length < 2000) {
-			await message.reply({
-				content: lmaod,
-				allowedMentions: { parse: [] }
+	// * Message was sent in the guild
+	if (!(message.channel instanceof Discord.DMChannel)) {
+		// * ayy => lmaoo
+		if (ayyRegex.test(message.content)) {
+			const lmaod = message.content.replaceAll(ayyRegex, (match) => {
+				let newMatch = match.replaceAll('y', 'o').replaceAll('Y', 'O');
+				newMatch = newMatch.replaceAll('a', 'lma').replaceAll('A', 'LMA');
+				return newMatch;
 			});
-		} else {
-			await message.reply('Looks like the resulting message is too long :/');
+
+			// * Check that the message isn't too long to be sent
+			if (lmaod.length < 2000) {
+				await message.reply({
+					content: lmaod,
+					allowedMentions: { parse: [] }
+				});
+			} else {
+				await message.reply('Looks like the resulting message is too long :/');
+			}
 		}
 
-		return;
-	}
+		// * Check if automatic help is disabled
+		const isHelpDisabled = await database.checkAutomaticHelpDisabled(message.guildId, message.member.id);
 
-	// * Check if automatic help is disabled
-	const isHelpDisabled = await database.checkAutomaticHelpDisabled(message.guildId, message.member.id);
+		if (!isHelpDisabled) {
+			// * Only do automatic help if not disabled
+			await tryAutomaticHelp(message);
+		}
 
-	if (isHelpDisabled) {
-		// * Bail if automatic help is disabled
-		return;
+		await networkDumpsUploader(message);
```

Comment:
This doesn't work over DMs, is this intentional?

Reply:
It is because of the way the bot is designed there's not a good way to get the guild ID from a DM channel. All of the settings for the bot are per-guild, since the bot is in multiple servers for testing purposes. It's possible to remove this and make it assume it's always in a single guild, but that's out of the scope of this PR imo